datadog_lambda/tracing.py

```diff
@@ -43,9 +43,15 @@
 
 logger = logging.getLogger(__name__)
 
+AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME"
+DD_SERVICE = "DD_SERVICE"
```

Comment:
Where is AWS_LAMBDA_FUNCTION_NAME documented? I couldn't find it on https://docs.datadoghq.com/serverless/aws_lambda/configuration/?tab=datadogcli which is in https://github.com/DataDog/datadog-lambda-python/blob/6f74908584dc8a937cfe026b4807b4a71dffc08e/README.md#L17

Reply:
I see DD_SERVICE mentioned there

Reply:
As per our conversation, https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime